app/platform/platform.c

```diff
@@ -361,6 +361,7 @@ uint32_t platform_uart_setup( unsigned id, uint32_t baud, int databits, int pari
     case BIT_RATE_4800:
     case BIT_RATE_9600:
     case BIT_RATE_19200:
+    case BIT_RATE_31250:
```

Comment:
You probably also want to add this bitrate to the uart_bitrates array. This will allow the get operation to return this bitrate.